packages/react-native-editor/CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ For each user feature we should also add a importance categorization label  to i
 
 * [***] Add support for cross-posting between sites
 * [***] Full-width and wide alignment support for Columns
+* [**] Add support for setting heading anchors
```

Comment:
We may need to move this note under `Unreleased` at this point.

Reply:
Good catch! Done.